sklearn/neighbors/_base.py

```diff
@@ -42,29 +42,30 @@
     kd_tree=KDTree.valid_metrics,
     # The following list comes from the
     # sklearn.metrics.pairwise doc string
-    brute=(
-        list(PAIRWISE_DISTANCE_FUNCTIONS.keys())
-        + [
-            "braycurtis",
-            "canberra",
-            "chebyshev",
-            "correlation",
-            "cosine",
-            "dice",
-            "hamming",
-            "jaccard",
-            "kulsinski",
-            "mahalanobis",
-            "matching",
-            "minkowski",
-            "rogerstanimoto",
-            "russellrao",
-            "seuclidean",
-            "sokalmichener",
-            "sokalsneath",
-            "sqeuclidean",
-            "yule",
-        ]
+    brute=list(
```

Comment:
I think the failures in the CI come from the parametrization over parameters that come from a set. According to https://github.com/pytest-dev/pytest-xdist/issues/598, we should use ``sorted(set)`` instead of ``list(set)``.
```suggestion
    brute=sorted(
```

Reply:
@jeremiedbb Thank you for your prompt comment!
I was just about to investigate the CI errors.
I will try.

Reply:
@jeremiedbb Ah. you have already fixed it. Thanks!